pkg/daemon/controller/controller.go

```diff
@@ -458,6 +465,25 @@ func (sc *ServiceController) resolvePowershellVariables(svc servicescm.Service)
 	return vars, nil
 }
 
+// startPeriodicServiceReconciliation initiates periodic polling to ensure that the instance is properly reconciled.
+func (sc *ServiceController) startPeriodicServiceReconciliation() {
+	go func() {
+		// Keeping this on the longer side, as each reconciliation requires running each service's powershell scripts
+		wait.PollInfinite(60*time.Second, func() (done bool, err error) {
+			select {
+			case <-sc.ctx.Done():
```

Comment:
I am noticing this just now. It is bad practice to add context to a struct. This is clearly called out in the [doc](https://pkg.go.dev/context):
>Do not store Contexts inside a struct type; instead, pass a Context explicitly to each function that needs it. The Context should be the first parameter, typically named ctx

Why are we doing this?

Reply:
I am fine doing it as a follow up PR.